src/test/java/teammates/test/BaseTestCaseWithSqlDatabaseAccess.java

```diff
@@ -47,20 +65,113 @@ protected void verifyEquals(BaseEntity expected, ApiOutput actual) {
             FeedbackQuestionDetails expectedQuestionDetails = expectedQuestion.getQuestionDetailsCopy();
             FeedbackQuestionData actualQuestion = (FeedbackQuestionData) actual;
             assertEquals(expectedQuestion.getQuestionNumber(), (Integer) actualQuestion.getQuestionNumber());
-            assertEquals(expectedQuestionDetails.getQuestionText(), actualQuestion.getQuestionBrief());
             assertEquals(expectedQuestion.getDescription(), actualQuestion.getQuestionDescription());
-            assertEquals(expectedQuestionDetails.getQuestionType(), actualQuestion.getQuestionType());
             assertEquals(expectedQuestion.getGiverType(), actualQuestion.getGiverType());
             assertEquals(expectedQuestion.getRecipientType(), actualQuestion.getRecipientType());
-            // TODO: compare the rest of the attributes D:
+            assertEquals(expectedQuestion.getNumOfEntitiesToGiveFeedbackTo(),
+                    actualQuestion.getCustomNumberOfEntitiesToGiveFeedbackTo());
+            assertEquals(expectedQuestionDetails.getQuestionText(), actualQuestion.getQuestionBrief());
+            assertEquals(expectedQuestionDetails.getQuestionType(), actualQuestion.getQuestionType());
```

Comment:
we should compare the json string of the question details, similar to feedbackresponse